### PR TITLE
Split text into multiple messages

### LINKED ISSUE #15: Use `telegram_bot/bot:_split_messages` for `update_hhh_message`
Currently the message is 2418/4096 characters long, we could run into problems here.

### PR DESCRIPTION
**Pull request by torbencarstens**
kicked from chat (reduced message count)
-> delete old unused messages

added to chat (increased message count)
-> send a new set of message

Closes #15

### CODE REVIEW

**Comment by torbencarstens** [comment]
Migrate all internal ids to int.

File: telegram_bot/bot.py [hunk 1]
```diff
@@ -135,15 +135,64 @@ def create_latest_change_text(chat: Chat, new_title: str, delete: bool = False)
 
         return change
 
-    def build_hhh_group_list_text(self) -> str:
-        text: str = ""
+    def build_hhh_group_list_text(self, prefix: str = "", suffix: str = "") -> List[str]:
+        """
+        For now, we'll assume that chats starting with the same letter will all fit into a single message
+        :param suffix:
+        :arg prefix: Will throw off the `deductable_per_chat` by `deductable_per_chat`.
```
Comment (BjoernPetersen) [inline telegram_bot/bot.py:142]: * Should use `:param`
* The description doesn't make much sense to me

File: telegram_bot/bot.py [hunk 2]
```diff
@@ -135,15 +135,64 @@ def create_latest_change_text(chat: Chat, new_title: str, delete: bool = False)
...
+        chats = [chat for _, chat in self.chats.items() if chat.title and chat.type != ChatType.PRIVATE]
+
+        messages = []
+        message = f"{prefix}\n" if prefix else ""
+        """
+        Telegram counts the character count after entity parsing.
+        i.e. <a href="https://example.com">A</a> should only be one character
+        We need this for the invite links
+        """
+        deductable_per_chat = 0
```
Comment (BjoernPetersen) [inline telegram_bot/bot.py:154]: This variable is never written to

**Review by BjoernPetersen — CHANGES_REQUESTED** [review]

File: telegram_bot/bot.py (hunk 2 above)
Comment (torbencarstens) [reply telegram_bot/bot.py:154]: `We need this for the invite links` (https://github.com/torbencarstens/hhh-diff-bot/issues/15#issuecomment-765799796)

File: telegram_bot/bot.py (hunk 1 above)
Comment (torbencarstens) [reply telegram_bot/bot.py:142]: The comment is wronng anyway, I initially wanted to calculate the length different than I do now. It doesn't affect anything anymore.

**Review by BjoernPetersen — APPROVED** [review]

**Comment by torbencarstens** [comment]
I've the feeling that there was a bug in this implementation, needs testing before a merge can happen.

**Pull request merged**